src/main/java/com/gluonhq/gradle/tasks/ConfigBuild.java

```diff
@@ -63,11 +64,38 @@
     }
 
     void configClient() {
-        clientConfig = new Config();
+        clientConfig = new Configuration();
         clientConfig.setGraalLibsVersion(clientExtension.getGraalLibsVersion());
         clientConfig.setJavaStaticSdkVersion(clientExtension.getJavaStaticSdkVersion());
         clientConfig.setJavafxStaticSdkVersion(clientExtension.getJavafxStaticSdkVersion());
-        clientConfig.setTarget(clientExtension.getTarget().toLowerCase(Locale.ROOT));
+
+        TargetTriplet hostTriplet = null, targetTriplet = null;
+        String target = clientExtension.getTarget().toLowerCase(Locale.ROOT);
+        switch (target) {
+            case Constants.TARGET_HOST:
+                String osname = System.getProperty("os.name", "Mac OS X").toLowerCase(Locale.ROOT);
+                if (osname.contains("mac")) {
+                    hostTriplet = new TargetTriplet(Constants.AMD64_ARCH, Constants.HOST_MAC, Constants.TARGET_MAC);
```

Comment:
probably better to split this: set host and target separately. Add a switch on os.name outside the target switch